subprojects/internal-integ-testing/src/main/groovy/org/gradle/integtests/fixtures/executer/AbstractGradleExecuter.java

```diff
@@ -101,6 +102,7 @@
         .displayName("Global services")
         .parent(newCommandLineProcessLogging())
         .parent(NativeServicesTestFixture.getInstance())
+        .parent(ValidationServicesFixture.getServices())
```

Comment:
Great that this works! Does this have any side-effect for the non-embedded tests?

Reply:
It doesn't, as far as I can tell. I wish it worked for non-embedded tests too, that would have been even better!